### PR TITLE
feat: add bk cc api

### PR DESCRIPTION
**Pull request by dellkeji**
变更点(Changes)
添加蓝鲸CC的api

### CODE REVIEW

**Comment by tencent-adm** [comment]
[CLA assistant check](https://cla.opensource.tencent.com/Tencent/bk-bcs-saas?pullRequest=1242) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.opensource.tencent.com/Tencent/bk-bcs-saas?pullRequest=1242) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.opensource.tencent.com/check/Tencent/bk-bcs-saas?pullRequest=1242) it.</sub>

File: bcs-app/backend/components/cc.py [hunk 1]
```diff
@@ -330,6 +337,66 @@ def cmdb_base_request(suffix_path, username, data, bk_supplier_account=None):
...
+
+@dataclass
+class PageData:
+    start: int = 0
+    limit: int = 200
+    sort: str = ""  # 排序字段
+
+
+class BkCCClient(BkApiClient):
+    def __init__(self, username: str, bk_supplier_account: str = DEFAULT_SUPPLIER_ACCOUNT):
```
Comment (piglei) [inline bcs-app/backend/components/cc.py:382]: 类型应该不太对，如果是 None 的话，类型应该是 Optional[str]，否则就不用 None，用 ''。

Comment (dellkeji) [reply bcs-app/backend/components/cc.py:382]: 需要支持None， 调整为Optional[str]=None

File: bcs-app/backend/components/cc.py [hunk 2]
```diff
@@ -330,6 +337,66 @@ def cmdb_base_request(suffix_path, username, data, bk_supplier_account=None):
...
+        self.host = host
+
+        # 请求地址
+        self.search_biz_url = f"{host}/{PREFIX_PATH}/v2/cc/search_business/"
+
+
+class BkCCAuth(AuthBase):
+    """用于蓝鲸配置平台接口的鉴权校验"""
+
+    def __init__(self, username: str, bk_supplier_account: str = None):
```
Comment (jamesgetx) [inline bcs-app/backend/components/cc.py:354]: 注解的问题

**Review by jamesgetx — APPROVED** [review]

**Pull request merged**